src/components/resourceBar/resourceInfoLabels.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright 2025 SAP SE
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from "react";
+import { CustomZones } from "../../lib/constants";
+
+export const UnknownAZLabel = (
+  <span data-testid="UnknownAZ">
+    This usage number accounts for assets in error states that are not associated with a real AZ.
+  </span>
+);
+
+export const CommittedUsageLabels = {
+  FULLY_UTILIZED: <span data-testid="CommittedUsage.FULLY_UTILIZED">Commitments are fully utilized.</span>,
+  UNUSED: (amount) => (
+    <span data-testid="CommittedUsage.UNUSED">
+      Unused commitments: <strong>{amount}</strong>.{" "}
+    </span>
+  ),
+  INVALID: (
+    <span data-testid="CommittedUsage.INVALID">
+      Displayed usage should not exceed commitments. Please create a support ticket.
```

Comment:
I find this message to be unintuitive, should this be 
```suggestion
      Displayed commitment usage should not exceed commitment amount. Please create a support ticket.
```